src/user-selection/components/users.css

```diff
@@ -18,16 +18,25 @@
 .userOption {
   height: 50px;
   width: 100%;
+  max-width: 100%;
   border: solid 1px black;
   border-bottom-width: 0px;
   display: flex;
-  justify-content: center;
   align-items: center;
   box-sizing: border-box;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
 }
+
 .userOption:hover {
   cursor: pointer;
 }
+
+.active:hover .removeUser {
+  display: block;
```

Comment:
Change this over to align with the message buttons.
Use `opacity` instead of `display`, and transition it over 200ms.